reconcile/utils/ocm/clusters.py

```diff
@@ -72,15 +73,7 @@ class ClusterDetails(BaseModel):
     found on the subscription of a cluster.
     """
 
-    subscription_labels: dict[str, OCMSubscriptionLabel]
-    organization_labels: dict[str, OCMOrganizationLabel]
-
-    def get_label(self, name: str) -> Optional[OCMLabel]:
-        if name in self.subscription_labels:
-            return self.subscription_labels[name]
-        if name in self.organization_labels:
-            return self.organization_labels[name]
-        return None
+    labels: LabelContainer
 
 
 def discover_clusters_by_labels(
```

Comment:
Some basic thoughts: How many API calls will this generate?  Do we perhaps need to add some caching? Or increase the pagesize to reduce API call count?